src/index.html

```diff
@@ -9,10 +9,33 @@
           maximum-scale=1.0,
           minimum-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Document</title>
-  <link rel="stylesheet" href="./style.css">
+  <title>Enclousers</title>
+  <link rel="stylesheet" href="style.css">
 </head>
 <body>
+  <div id="qa-block" class="box box__color--first">
+    <div class="box box__color--second">
```

Comment:
ok for now, but block cannot be BEM block and self child(element) at the same time

Reply:
and how, then, should I be done?

Reply:
don't use class "box" in line 17?